lms/djangoapps/edxnotes/tests.py

```diff
@@ -525,43 +525,6 @@ def test_search_empty_collection(self, mock_get):
             json.loads(helpers.get_notes(self.request, self.course))
         )
 
-    def test_preprocess_collection_escaping(self):
```

Comment:
Actually, I meant a Python test to show how tag "First" and "Second" return from ES search as "First" and "{elasticsearch_highlight_start}Second{elasticsearch_highlight_end}"?